src/index.html

```diff
@@ -257,7 +257,91 @@ <h2>QUICK ID</h2>
 
         <section>
           <h2>CONTACT ME</h2>
-          <!-- Form Related Content for Lesson 2 -->
+        </section>
+      </div> -->
+
+      <div id="info" class="info-container">
+        <section id="info-quick-id" class="info-section">
+          <h2 class="info-title">QUICK ID</h2>
+          <div class="data-border"></div>
```

Comment:
Better to use the CSS sudo element to achieve this, e.g.
```
h2:after {
  content: '';
  width: ...
  height: ...
  background-color:...
}
```
You are introducing redundant no meaningful div here.